service.go

```diff
@@ -89,19 +108,25 @@ func (ts *telemetryService) ServeUI(ctx context.Context, filters TelemetryFilter
 		to = filters.To.Format(time.DateOnly)
 	}
 	data := struct {
-		Countries     string
-		NoDeployments int
-		NoCountries   int
-		MapData       string
-		From          string
-		To            string
+		Countries       string
+		Cities          string
+		FilterCountries []CountrySummary
+		FilterCities    []CitySummary
+		NoDeployments   int
+		NoCountries     int
+		MapData         string
+		From            string
+		To              string
 	}{
-		Countries:     string(countries),
-		NoDeployments: summary.TotalDeployments,
-		NoCountries:   len(summary.Countries),
-		MapData:       string(pg),
-		From:          from,
-		To:            to,
+		Countries:       string(countries),
+		Cities:          string(cities),
```

Comment:
This seems unnecessary since I don't see where this data is being used in your html template